app/views/users/_card.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="card shadow rounded-lg">
+  <div class="card-body">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs">
+          <img src="//lh3.googleusercontent.com/-6V8xOA6M7BA/AAAAAAAAAAI/AAAAAAAAAAA/rzlHcD0KYwo/photo.jpg?sz=120" class="rounded-circle" style="width: 50px">
+        </div>
+        <a class="col-8" href=<%= user_path(user) %>>
+          <h5 class="card-title"><%= user.name %></h5>
+          <small class="card-link"><i><%= user.email %></i></small>
+        </a>
+        <% if mode == :add %>
+          <div class="col-1">
+            <%# fa_icon helper: https://github.com/bokmann/font-awesome-rails %>
+            <%= link_to (fa_icon 'plus-circle', title: I18n.t('user.contact_request.send')), user_contact_requests_path(user), id: user.id, class: "fa-2x"%>
+          </div>
+        <% end %>
+      </div>
+      <% if mode == :overview %>
+        <div class="row top-buffer">
+          <% if user.social_accounts.any? %>
```

Comment:
shouldn't be necessary as the block given to each is not executed if there are no elements in the collection